.devcontainer/devcontainer.json

```diff
@@ -12,35 +12,37 @@
 			"NODE_VERSION": "lts/*"
 		}
 	},
-
```

Comment:
These changes relate to the config for using a GitHub Codespace or VS Code remote as an IDE - see https://github.com/agilepathway/gauge-jira/pull/4